code/src/index.css

```diff
@@ -1,13 +1,84 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@100&display=swap");
```

Comment:
The styling is nice, I like the emojis. One suggestion is to put some padding to the inputs to increase accessibility. They can get quite small, especially on mobile.